woocommerce/storefront
# Sticky Add To Cart

Integrates [Storefront Sticky Add to Cart](https://pt.wordpress.org/plugins/storefront-sticky-add-to-cart/) with a few improvements.

* No jQuery
* Works will all product types:
  **Single** - Adds product to cart
  **External** - Go to site
  **Variable/Grouped/Others** - Scroll to form

To test:

1. Checkout this branch
2. Go to Single Product page
3. Browse past the product description area
4. You should see:

<img width="1188" alt="screen shot 2018-03-13 at 19 57 22" src="https://user-images.githubusercontent.com/1177726/37366595-d10e01d4-26f8-11e8-9c74-6c6f811aad26.png">

Closes #765.
